tests/e2e/default/more-options-menu/online-user/permissions-disabled.wdio-spec.js

```diff
@@ -62,72 +54,90 @@ describe('- permissions disabled', () => {
 
     after(async () => await utils.revertSettings(true));
 
-    it(' - Contact Tab - contact selected', async () => {
+    it('should hide the \'export\' option and ' +
+      'enable the \'edit\' and \'delete\' options when a contact is opened', async () => {
       await commonPage.goToPeople(contact._id);
       await commonPage.openMoreOptionsMenu();
       expect(await commonPage.isMenuOptionVisible('export', 'contacts')).to.be.false;
       expect(await commonPage.isMenuOptionEnabled('edit', 'contacts')).to.be.true;
       expect(await commonPage.isMenuOptionEnabled('delete', 'contacts')).to.be.true;
     });
 
-    it('- Report Tab - report selected', async () => {
+    it('should hide the \'export\' option and ' +
+      'enable the \'edit\', \'delete\' and \'review\' options when a report is opened', async () => {
       await commonPage.goToReports();
       expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;
       (await reportPage.firstReport()).click();
       await commonPage.openMoreOptionsMenu();
       expect(await commonPage.isMenuOptionVisible('export', 'reports')).to.be.false;
+      expect(await commonPage.isMenuOptionEnabled('edit', 'reports')).to.be.true;
       expect(await commonPage.isMenuOptionEnabled('delete', 'reports')).to.be.true;
+      expect(await commonPage.isMenuOptionEnabled('review', 'report')).to.be.true;
     });
 
-    it('- Message tab', async () => {
+    it('should hide the kebab menu the it is on the Message tab', async () => {
```

Comment:
Can we make the description of this test better?

Reply:
Updated, thank you.